Sources/IO/MediaLink.swift

```diff
@@ -78,6 +78,7 @@ final class MediaLink<T: MediaLinkDelegate> {
                 isBuffering = false
             }
         }
+        mach_absolute_time()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/shogo4405/HaishinKit.swift/blob/7fedc52c24ef08832a21716080a8697f62f01528/Sources/IO/MediaLink.swift#L81"></span>The 'mach_absolute_time' is the require reason api. Please check privacy manifest files.</td>
    </tr>
  </tbody>
</table>